src/ci/docker-compose/docker-compose-horicromtal.yml

```diff
@@ -34,6 +34,8 @@ services:
       - CROMWELL_BUILD_PAPI_DOCKER_IMAGE_DRS
       - CROMWELL_BUILD_PAPI_ENDPOINT_URL
       - CROMWELL_BUILD_RESOURCES_DIRECTORY
```

Comment:
nit: is it possible to add some sort of comment explaining the additional env vars here?

Reply:
Yup I can add a comment here.